tools/src/main/java/com/graphhopper/tools/Measurement.java

```diff
@@ -242,7 +242,7 @@ protected DataReader importData() throws IOException {
                 System.gc();
                 if (!hopper.getCHPreparationHandler().getNodeBasedCHConfigs().isEmpty()) {
                     CHConfig chConfig = hopper.getCHPreparationHandler().getNodeBasedCHConfigs().get(0);
-                    CHGraph lg = g.getCHGraph(chConfig);
+                    CHGraph lg = g.getCHGraph(chConfig.getName());
```

Comment:
I changed these, because getting a CH graph should be done by passing its name, not the entire `CHConfig` (which I want to get rid of anyway).